SNIPS/snip-75.md

```diff
@@ -0,0 +1,213 @@
+---
+snip: 75
+title: Starknet Security Council (SSC)
+author: Elias Tazartes <@Eikix>, Mattéo Georges <@mattegoat>, Patrícia Gil de Brolezzi <@pat_zip>, luiz-lvj <lvj253@gmail.com>, Luca Donno <@lucadonnoh>, Nicolas Consigny <@nconsigny>, et al.
```

Comment:
main author should be whoever designed the document from SW / SNF since I was not involved

Reply:
I merged both document trying to keep your doc as a basis